python/foglamp/plugins/south/wind_turbine/wind_turbine.py

```diff
@@ -229,13 +230,14 @@ def plugin_info():
     """
     return {
         'name': 'wind_turbine Poll Plugin',
-        'version': '2.0.0',
+        'version': '1.5.0',
```

Comment:
We should update it

Reply:
@Singhal-Vaibhav  could you please check and confirm, we are replacing all these for all python plugins?  I think C plugins does not need this treatment /cc @MarkRiddoch